docs/blocks/rCode.md

```diff
@@ -0,0 +1,62 @@
+# R Code
+
+It allows to run R code in the browser.
+
+This block allows multiple inputs and outputs. This makes it interesting to tie the behaviors of different blocks together.
+
+To set the inputs and outputs for the script, select "open extended view" which opens a popup.
+
+## Output
+
+As defined in the extended but limited to the objects that can be converted to JavaScript (see [here](https://docs.r-wasm.org/webr/latest/convert-r-to-js.html){:target="\_blank"}).</br>
```

Comment:
somehow I am tripping over this sentence.
maybe we could say: `Currently not all R objects support conversion to output. (see ... )`